codepropertygraph/src/main/scala/io/shiftleft/codepropertygraph/cpgloading/CpgOverlayLoader.scala

```diff
@@ -46,17 +48,35 @@ private[cpgloading] object CpgOverlayLoader {
 private class CpgOverlayApplier(graph: ScalaGraph) {
 
   private val overlayNodeIdToSrcGraphNode: mutable.HashMap[Long, Vertex] = mutable.HashMap()
+  private var inverseBuilder: DiffGraph.InverseBuilder = _
```

Comment:
`var =:= a few months in hell`
just kidding

Reply:
Could the `InverseBuilder` be passed as an argument to the functions that use it?

Reply:
@GlassAndOneHalf yes, and would avoid var, so I'll do that